iina/ShooterSubtitle.swift

```diff
@@ -10,6 +10,8 @@ import Foundation
 import Just
 import PromiseKit
 
+fileprivate let subsystem = Logger.Subsystem(rawValue: "shooter")
```

Comment:
I know this is how other code does it, but we should really move to making these extensions on `Logger` (preferably, one that is namespaced) rather than just a global at the top of the file. For example:

```swift
extension Logger.Sub {
	static let subsystem = Logger.Subsytem(rawValue: "sub.shooter")
}
```

Reply:
Will fix.

Reply:
Need some help with this one. I'm running into "invalid redeclaration" errors if I use this pattern in more than one place.

Reply:
This is what worked.

In OnlineSubtitle, the parent class, I'm defining the namespace and the subsystem for that class:
```swift
extension Logger {
  struct Sub {
    static let onlinesub = Subsystem(rawValue: "onlinesub")
  }
}
```

In OpenSubSubtitle I can then code it like shown above:
```swift
extension Logger.Sub {
  static let opensub = Logger.Subsystem(rawValue: "sub.shooter")
}
```

And of course all references must then be fully qualified:
```swift
Logger.log("OpenSub: heartbeat ok", subsystem: Logger.Sub.opensub)
```

Reply:
Fixed.